docs/setup/INSTALLATION.md

```
@@ -0,0 +1,187 @@
+
+### Docker Compose Profiles
+
+Start specific services:
+
+```bash
+# Gateway only (no translate services)
+make gateway-only
+
+# Full stack (default)
+make start
+
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)